src/codemagic/tools/app_store_connect.py

```diff
@@ -148,6 +154,72 @@ def _delete_resource(self, resource_manager, resource_id: ResourceId, ignore_not
             else:
                 raise AppStoreConnectError(str(api_error))
 
+    @cli.action('list-testflight-builds',
+                BuildArgument.APPLICATION_ID_OPTIONAL,
+                BuildArgument.EXPIRED,
+                BuildArgument.NOT_EXPIRED,
+                BuildArgument.BUILD_ID_OPTIONAL,
+                BuildArgument.PRE_RELEASE_VERSION,
+                BuildArgument.PROCESSING_STATE,
+                BuildArgument.BUILD_NUMBER,
+                BuildArgument.ORDERING,
+                BuildArgument.REVERSE)
+    def list_testflight_builds(self,
+                              application_id: Optional[Union[str, ResourceId]] = None,
+                              expired: Optional[bool] = None,
+                              not_expired: Optional[bool] = None,
+                              build_id: Optional[Union[str, ResourceId]] = None,
+                              pre_release_version: Optional[str] = None,
+                              processing_state: Optional[BuildProcessingState] = None,
+                              build_number: Optional[str] = None,
+                              ordering: Optional[BuildOrdering] = None,
```

Comment:
As stated elsewhere, I'm not convinced that we need ordering and if we do, then it should be implemented more generally for all `list_*` tasks.